README.md

```diff
@@ -125,15 +125,15 @@ Thank you for downloading and using the PrestaShop Open Source e-commerce soluti
 [forums]: https://www.prestashop.com/forums/
 [user-doc]: http://doc.prestashop.com
 [contributing-md]: CONTRIBUTING.md
-[contributing-tutorial]: http://doc.prestashop.com/display/PS16/Contributing+to+PrestaShop
+[contributing-tutorial]: http://doc.prestashop.com/display/PS17/Contributing+to+PrestaShop
 [crowdin]: https://crowdin.net/project/prestashop-official
 [getting-started]: http://doc.prestashop.com/display/PS17/Getting+Started
 [user-guide]: http://doc.prestashop.com/display/PS17/User+Guide
-[updating-guide]: http://doc.prestashop.com/display/PS16/Updating+PrestaShop
-[merchant-guide]: http://doc.prestashop.com/display/PS16/Merchant%27s+Guide
+[updating-guide]: http://doc.prestashop.com/display/PS17/Updating+PrestaShop
+[merchant-guide]: http://doc.prestashop.com/display/PS17/Merchant%27s+Guide
 [faq-17]: http://build.prestashop.com/news/prestashop-1-7-faq/
-[troubleshooting]: http://doc.prestashop.com/display/PS16/Troubleshooting
-[sysadmin-guide]: http://doc.prestashop.com/display/PS16/System+Administrator+Guide
+[troubleshooting]: http://doc.prestashop.com/display/PS17/Troubleshooting
+[sysadmin-guide]: http://doc.prestashop.com/display/PS17/System+Administrator+Guide
```

Comment:
Have you checked the different URLs, this one for instance brings to a "Page not found".

The actual URLs should probably be in the devdocs.

Reply:
My bad, checked by opening 5 tabs and I saw the page titles load so I thought they were not 404

JIRA 404 page actually parses the URL to generate the title 🤣